solidity/contracts/interfaces/IInterchainSecurityModule.sol

```diff
@@ -8,7 +8,8 @@ interface IInterchainSecurityModule {
         AGGREGATION,
         LEGACY_MULTISIG,
         MERKLE_ROOT_MULTISIG,
-        MESSAGE_ID_MULTISIG
+        MESSAGE_ID_MULTISIG,
+        OPTIMISM
```

Comment:
did we settle on multiple types mapping to null? I remember being a proponent of a single type but actually is nice to be able to modify independently